backend/fms_core/migrations/0027_v3_5_0.py

```diff
@@ -0,0 +1,127 @@
+# Generated by Django 3.1 on 2021-10-21 21:20
+
+from django.conf import settings
+from django.db import migrations, models
+from django.contrib.auth.models import User
+import reversion
+
+ADMIN_USERNAME = 'biobankadmin'
+
+
+def create_mgi_T7_related_objects(apps, schema_editor):
+    Platform = apps.get_model("fms_core", "Platform")
+    InstrumentType = apps.get_model("fms_core", "InstrumentType")
+    Instrument = apps.get_model("fms_core", "Instrument")
+    RunType = apps.get_model("fms_core", "RunType")
+    Protocol = apps.get_model("fms_core", "Protocol")
+    PropertyType = apps.get_model("fms_core", "PropertyType")
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        admin_user_id = admin_user.id
+
+        reversion.set_comment("Create objects related to MGI T7 experiment")
+        reversion.set_user(admin_user)
+
+        # Platform and InstrumentType already created for MGI T7
+        platform = Platform.objects.get(name="DNBSEQ")
+
+        # Instrument dictionary {NAME: TYPE} for creation
+        INSTRUMENTS = {
+            "01-Marie Curie": "DNBSEQ-G400",
+            "02-Frida Kahlo": "DNBSEQ-G400",
+            "03-Jennifer Doudna": "DNBSEQ-T7",
+        }
+        for name in INSTRUMENTS.keys():
+            it = InstrumentType.objects.get(type=INSTRUMENTS[name])
+            i = Instrument.objects.create(name=name,
+                                          type=it,
+                                          created_by_id=admin_user_id,
+                                          updated_by_id=admin_user_id)
+            reversion.add_to_revision(i)
+
+        # Create PropertyType and Protocols
+        PROPERTY_TYPES_BY_PROTOCOL = {
+            "DNBSEQ Preparation": [("Experiment Name", "str"),
+                                   ("Flowcell Lot", "str"),
+                                   ("Loading Method", "str"),
+                                   ("Sequencer Side", "str"),
+                                   ("Sequencer Kit Used", "str"),
+                                   ("Sequencer Kit Lot", "str"),
+                                   ("Load DNB Cartridge Lot", "str"),
+                                   ("Primer Kit", "str"),
```

Comment:
Not sure if you are missing "Primer Kit Lot" from the Template or there's a reason why we don't include it.